jobs/integr8ly/ocp4/ocp4-install.yaml

```diff
@@ -67,51 +75,67 @@
             default: 'integreatly-qe@redhat.com'
             description: "Whitespace- or comma-separated list of recipient addresses"
     dsl: |
-        Boolean cleanupAfterward = params.CLEANUP_AFTERWARD
+        #!groovy
         Boolean cleanupBeforehand = params.CLEANUP_BEFOREHAND
+        Boolean cleanupAfterward = params.CLEANUP_AFTERWARD
         Boolean selfSignedCerts = params.SELF_SIGNED_CERTS
+        Boolean isPds = params.PDS_CLUSTER
 
         String registryNamespace = 'integreatly'
         if (params.REGISTRY_NAMESPACE) {
             registryNamespace = params.REGISTRY_NAMESPACE
         }
 
-        String clusterAPI = "https://api.cluster-${RHPDS_CITY}.${RHPDS_CITY}.openshiftworkshop.com:6443"
+        String clusterSuffix = isPds ? "cluster-${CLUSTER_NAME}.${CLUSTER_NAME}.${CLUSTER_DOMAIN}" : "${CLUSTER_NAME}.${CLUSTER_DOMAIN}"
+        
+        String clusterAPI = "https://api.${clusterSuffix}:6443"
+
+        String adminUsername = isPds ? params.ADMIN : "kubeadmin"
+        String githubSecretId = "77b67c9d10dbc6b65c76" // secretId of a new Oauth app created in fh-automatron GH account
 
         // by default build number is used as namespacePrefix to prevent clashes with previous Integreatly Operator deployments
         String namespacePrefix = BUILD_NUMBER
         if (NAMESPACE_PREFIX) {
             namespacePrefix = NAMESPACE_PREFIX
         }
-        String namespaceName = namespacePrefix + '-integreatly-operator'
+        String namespaceName = namespacePrefix + '-integreatly-operator' 
 
+        String noConfigFilesError = "[ERROR] No configuration files found for ${CLUSTER_NAME}"
         def err = null
 
         node('cirhos_rhel7') {
-            timeout(30) { ansiColor('gnome-terminal') { timestamps {
+            timeout(35) { ansiColor('gnome-terminal') { timestamps {
                 try {
+                    currentBuild.displayName = "${currentBuild.displayName} ${CLUSTER_NAME}"
 
-                    stage('Install oc cli version 4') {
+                    stage('Login to OpenShift 4 cluster') {
                         withCredentials([string(credentialsId: "${ADMIN_PASSWORD_CREDENTIALS}", variable: 'ADMIN_PASSWORD')]) {
-                            sh(
-                                returnStdout: false,
-                                script: """
-                                    wget -q https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.1.8/openshift-client-linux-4.1.8.tar.gz
-
-                                    tar xvzf openshift-client-linux-4.1.8.tar.gz
-
-                                    ./oc login ${clusterAPI} --insecure-skip-tls-verify=true -u ${ADMIN} -p ${ADMIN_PASSWORD}
-                                    ./oc version
-                                """
-                            )
```

Comment:
Just a question, you removed this and just use oc v3 or oc v4 is somehow being used?

Reply:
oc v3 is used for now. There is a ticket to enable our slaves to use either of them: https://issues.jboss.org/browse/INTLY-3193